assertion/function/assertiontree/var_assertion_node.go

```diff
@@ -63,14 +63,14 @@ func (v *varAssertionNode) DefaultTrigger() annotation.ProducingAnnotationTrigge
 	// if `v` is a struct (e.g., var s S), not a struct pointer, then analyze it for its fields. Note that here we don't
 	// want to analyze fields of an unassigned struct pointer, since at this point the pointer itself is nil.
 	// TODO: below logic won't be required once we standardize the expression `var s S` by replacing it with `S{}` in the
-	//  preprocessing phase after  is implemented
-	if v.Root().functionContext.isDepthOneFieldCheck() {
-		if !util.TypeIsDeeplyPtr(v.decl.Type()) {
-			if structType := util.TypeAsDeeplyStruct(v.decl.Type()); structType != nil {
-				builtExpr := v.BuildExpr(v.Root().Pass(), nil)
+	//  preprocessing phase
+	if !util.TypeIsDeeplyPtr(v.decl.Type()) {
+		if structType := util.TypeAsDeeplyStruct(v.decl.Type()); structType != nil {
+			builtExpr := v.BuildExpr(v.Root().Pass(), nil)
```

Comment:
Should we move this inside the if check below (or just remove this temp var since the function call is clear enough)?

Reply:
Ah, good point. Done.